lib/Dancer2/Serializer/JSON.pm

```diff
@@ -79,3 +79,48 @@ Serializes a Perl data structure into a JSON string.
 =method deserialize($content)
 
 Deserializes a JSON string into a Perl data structure.
+
+
+
+=head2 Configuring the JSON Serializer using "set engines"
+
+The JSON serializer optoins can be configured via "set engines". The most
```

Comment:
Rather than wrapping set engines in quotes, wrap it in `C<>`. `perldoc` will know to format `C<set engines>` as code when it sees it.